apps/client/app/dashboard/page.tsx

```diff
@@ -18,7 +18,6 @@ import DashboardTop from '@/components/ui/Dashboard/DashboardTop'
 import DashboardContent from '@/components/ui/Dashboard/DashboardContent'
 import NavBar from '@/components/ui/Dashboard/Navbar'
```

Comment:
i think this is no longer being used, right?

Reply:
Resolved.
Component is nice and shiny.